lib/cuvva-error.js

```diff
@@ -15,7 +15,9 @@ function CuvvaError(code, reasons, meta) {
 	}
 
 	CuvvaError.super_(code);
-	Error.captureStackTrace(this, this.constructor);
+
+	if (Error.captureStackTrace)
+		Error.captureStackTrace(this, this.constructor);
```

Comment:
We should move to using the raw `Error.stack` instead of `captureStackTrace` (which is more V8 specific and not widely supported)

```suggestion
	if (Error.stack)
		this.stack = Error.stack;
```

`captureStackTrace` doesn't seem to do more than setting the `.stack` variable ourselves. It does support hiding some calls from the stack trace (anything from `this.constructor` in our case). Moving to the raw `.stack` call seems like a decent solution to get a wider supported stack trace. 

https://nodejs.org/api/errors.html#errors_error_capturestacktrace_targetobject_constructoropt

Thoughts?